achievements/templates/achievements/view-category.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'achievements/base.html' %}
+
+{% block content %}
+  <h3>Name:</h3>
+  <p>{{name}}</p>
+
+  <h3>Description:</h3>
+  <p>{{description}}</p>
+{% endblock %}
```

Comment:
(Future) improvement: list achievements in this category

Reply:
That is indeed something that I want to add in the next issue, where members will also be able to be linked to achievements and view their achievements